apps/judicial-system/backend/src/app/modules/case-table/caseTable.cellGenerators.ts

```diff
@@ -278,20 +307,21 @@ const generateRequestCaseType = (
   return [formatted, parentCaseId ? 'Framlenging' : ''].filter(Boolean)
 }
 
-const caseNumber: CaseTableCellGenerator = {
+const caseNumber: CaseTableCellGenerator<StringGroupValue> = {
   attributes: ['policeCaseNumbers', 'courtCaseNumber', 'appealCaseNumber'],
-  generate: (c: Case): StringGroupValue => ({
-    s: [
-      c.appealCaseNumber ?? '',
-      c.courtCaseNumber ?? '',
-      c.policeCaseNumbers.length > 1
-        ? `${c.policeCaseNumbers[0]} +${c.policeCaseNumbers.length - 1}`
-        : c.policeCaseNumbers[0],
-    ],
-  }),
-}
-
-const defendants: CaseTableCellGenerator = {
+  generate: (c: Case): CaseTableCell<StringGroupValue> =>
+    generateCell({
+      strList: [
+        c.appealCaseNumber ?? '',
+        c.courtCaseNumber ?? '',
+        c.policeCaseNumbers.length > 1
+          ? `${c.policeCaseNumbers[0]} +${c.policeCaseNumbers.length - 1}`
```

Comment:
We are using similar logic formatting in other places when there are more values. What do you think about extracting this as a helper?

```
const getAdditionalCellValuesFormat = (values: string[]) =>
  values.length > 1 ? `${values[0]} +${values.length - 1}` : values[0]

```

Reply:
I like it, but usually don't bother until we have a third or fourth instance. But if you like, you can implement it in a later PR.